packages/proto-signing/src/registry.spec.ts

```diff
@@ -51,34 +49,35 @@ describe("registry demo", () => {
     expect(msgSendDecoded.amount).toEqual(msgSend.amount);
   });
 
-  it("works with a custom msg", () => {
-    const typeUrl = "/demo.MsgDemo";
-    const registry = new Registry([[typeUrl, MsgDemoType]]);
-    const MsgDemo = registry.lookupType(typeUrl)!;
+  // TODO: Can't autogenerate types from TS code using ts-proto
```

Comment:
`MsgDemoType` is created at runtime using `protobuf` only:

```ts
import { Field, Root, Type } from "protobufjs";

export const MsgDemo = new Type("MsgDemo").add(new Field("example", 1, "string"));

const root = new Root().define("demo").add(MsgDemo);

export default root;
```

This is used by the Starport team: https://twitter.com/fadeev/status/1345006725586440192

Shouldn't it be possible to use them the same way as before via one of those constructors?

```ts
/** Reflected message type. */
export class Type extends NamespaceBase {
    // …

    /**
     * Creates a message type from a message type descriptor.
     * @param name Message name
     * @param json Message type descriptor
     * @returns Created message type
     */
    public static fromJSON(name: string, json: IType): Type;

    // …

    /**
     * Creates a new message of this type using the specified properties.
     * @param [properties] Properties to set
     * @returns Message instance
     */
    public create(properties?: { [k: string]: any }): Message<{}>;

    // …
}
```